api/lql_data_sources.go

```diff
@@ -0,0 +1,33 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+type LQLDataSourcesResponse struct {
+	Data    []string `json:"data"`
+	Ok      bool     `json:"ok"`
+	Message string   `json:"message"`
+}
+
+func (svc *LQLService) DataSources() (
+	response LQLDataSourcesResponse,
+	err error,
+) {
+	err = svc.client.RequestDecoder("GET", ApiLQLDataSources, nil, &response)
```

Comment:
```suggestion
	err = svc.client.RequestDecoder("GET", apiLQLDataSources, nil, &response)
```

Reply:
So that I can access this via `api.ApiLQLDataSources` in my test file:

```
func TestDataSourcesMethod(t *testing.T) {
	fakeServer := lacework.MockServer()
	fakeServer.MockAPI(
		api.ApiLQLDataSources,
```